### DISCUSSION TITLE
Problems with build

### DISCUSSION THREAD

**Discussion by kunkanader** [Floorp-Projects/Floorp#836] [Q&A]
I run into a problem when tried to build floorp myself using this manual (win 10 x64):
https://github.com/Floorp-Projects/Floorp/wiki/developers-building-windows
```
No such file or directory: 'C:/Users/*/Floorp/floorp/moz.configure
```
on "./mach bootstrap" step.
It appears to me that it tries to find this file inside Floorp/floorp instead of Floorp. 
I am a codelet so I apologize if the issue is trivial and I'm being retarded. This is the first time I'm trying to build a firefox based browser, i.e I never even built firefox before.

**Comment by typeling1578** [comment]
Did you set up a git submodule?
```
git submodule update --init --recursive
```

**Reply by kunkanader** [reply]
Turns out I was expected to execute this command inside floorp folder, and it didn't work at first. Built it successfully now, thanks.
